components/QuickStartMenu.tsx

```diff
@@ -24,6 +67,62 @@ const QuickStartMenu = ({ className }: QuickStartMenuProps): JSX.Element => {
           </g>
         </svg>
       </button>
+      {/* NOTE:: Dropdown currently absolutely positioned. Revist if used in place other than the navigation */}
+      <div
```

Comment:
You'll want to add `id` elements to the `button` and this wrapping `<div>`, use `react-id-generator` - you can find an example in `SearchBox.tsx`. 

Then you'll also want `aria-labelledby` on this element, pointing to the `id` of the `<button>` element.